lab-andrew/gulpfile.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+const gulp = require('gulp');
+const mocha = require('gulp-mocha');
+// const eslint = require('gulp-eslint');
+
+
+
+gulp.task('test', function() {
+  return gulp.src(__dirname + '/test/*.js')
+  .pipe(mocha());
+});
+// gulp.task('lint', function() {
+//   return gulp.src(['**/*.js', '!node_modules/**'])
+//   .pipe(eslint()).pipe(eslint.format());
+// });
+var watcher = gulp.watch(['**/*.js','!node_modules**'], ['test']);
```

Comment:
I suggest you declare and call your watch task differently, see my comment in Slack or talk to me if you have any questions about what I mean.